src/EditorFeatures/Core/InlineRename/AbstractEditorInlineRenameService.InlineRenameLocationSet.cs

```diff
@@ -7,22 +7,21 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Options;
 using Microsoft.CodeAnalysis.Rename;
-using Roslyn.Utilities;
+using Microsoft.CodeAnalysis.Rename.ConflictEngine;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.InlineRename
 {
     internal abstract partial class AbstractEditorInlineRenameService
     {
         private class InlineRenameLocationSet : IInlineRenameLocationSet
         {
-            private readonly RenameLocations _renameLocationSet;
+            private readonly LightweightRenameLocations _renameLocationSet;
```

Comment:
inline rename now only holds onto "LightweightRenameLocations" so it can cheaply call back into OOP and so no local symbols/compilations are created for it.

Reply:
LightweightRenameLocations still holds on ISymbol. Why is that?

Reply:
for ease of making this PR.  it doesn't ever actually *marshall* that symbol.  it's just a pointer *on the host* to the original symbol that hte host determined it wants to rename.